in_text/text_00_parent.yml

```diff
@@ -16,22 +16,22 @@ abstract: >-
+  <br/>The data are organized into 4 child folders: 1) waterbody information, 2) model driver data, 3) model configurations, 4) model predictions </p>
   <ol>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a285d82ce38aaa244912e">1) Waterbody Information</a> - One shapefile of polylines for 70 river segments in this study, and one shapefile metadata file of reservoir polygons for the Pepacton and Cannonsville reservoirs</li>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a287382ce38aaa2449131">2) Model Driver Data </a> - Data used to drive predictive models (daily meteorology for river reaches and reservoirs, observed reservoir diversions and releases)</li>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a288982ce38aaa2449133">3) Model Configurations </a> - Model parameters and metadata used to configure GLM 3.1 reservoir models </li>
-  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a28a782ce38aaa2449137">4) Model Predictions </a> - Temperature predictions data files, including GLM 3.1 predictions of ourflow and water temperature for reservoir outflow reaches, stream temperature predictions from the distance-weighted-average lotic-lentic input network, and 7-day ahead deep learning water temperature forecasts at 5 priority sites. </li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a285d82ce38aaa244912e">Waterbody Information</a> - One shapefile of polylines for 70 river segments in this study, and one shapefile of reservoir polygons for the Pepacton and Cannonsville reservoirs</li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a287382ce38aaa2449131">Model Driver Data </a> - Data used to drive predictive models (daily meteorology for river reaches and reservoirs, observed reservoir diversions and releases)</li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a288982ce38aaa2449133">Model Configurations </a> - Model parameters and metadata used to configure GLM 3.1 reservoir models </li>
+  <li><a href="https://www.sciencebase.gov/catalog/item/5f6a28a782ce38aaa2449137">Model Predictions </a> - Temperature predictions data files, including GLM 3.1 predictions of outflow and water temperature for reservoir outflow reaches, stream temperature predictions from the distance-weighted-average lotic-lentic input network, and 7-day ahead deep learning water temperature forecasts at 5 priority sites. </li>
   <br/>
   <p>This research was funded by the USGS.</p>
 
-build-environment: Multiple computer operating systems were used to generate these data, including Windows, linux, OSX. The open source languages R and Python were used on all systems.
+build-environment: Multiple computer operating systems were used to generate these data, including Windows, Linux, and OSX. The open source languages R and Python were used on all systems.
```

Comment:
I'm a little confused about why the parent build environment says this, and then the child items describe the R packages used to build the data release. Why aren't they all the same? Or even the opposite would make more sense to me (parent has info about data release, children have info about how the actual data was generated)